Formula/corsixth.rb

```diff
@@ -24,6 +24,10 @@ class Corsixth < Formula
   depends_on "sdl2"
   depends_on "sdl2_mixer"
 
+  on_linux do
```

Comment:
I tried running on Docker via XQuartz and it looks like it may need OpenGL (e.g. `mesa`) at runtime:
```
...rew/Cellar/corsixth/0.65.1_1/share/corsix-th/Lua/app.lua:164: Could not initialize OpenGL / GLES library
```